db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_06_22_094606) do
```

Comment:
- [ ] I don't think the schema file is supposed to be altered. This particular block is missing in your schema file. 
![Screenshot from 2023-06-23 23-30-52](https://github.com/Gmanlove/blog-app/assets/63087491/ebed60cd-4167-49d5-8d7d-f46462c3ad17)